src/main.rs

```diff
@@ -1,24 +1,47 @@
 mod config;
-use actix_web::{get, App, HttpServer, Responder};
+use actix_web::{web, App, HttpServer, Responder};
 use anyhow::Result as AnyhowResult;
 use tracing_actix_web::TracingLogger;
 
-#[get("/")]
```

Comment:
I liked having this annotation here, but did some transformations behind the scene that make unit tests a little annoying.

We can put them back at some point if it gets annoying but for now I figured it'd be simpler to just set things up this way